keras_cv/layers/__init__.py

```diff
@@ -19,6 +19,13 @@
 
 from keras_cv.layers.feature_pyramid import FeaturePyramid
 from keras_cv.layers.fusedmbconv import FusedMBConvBlock
+from keras_cv.layers.maxvit_layers import GridPartitioning
+from keras_cv.layers.maxvit_layers import MaxViTBlock
+from keras_cv.layers.maxvit_layers import MaxViTStem
+from keras_cv.layers.maxvit_layers import RelativeMultiHeadAttention
+from keras_cv.layers.maxvit_layers import UnGridPartitioning
+from keras_cv.layers.maxvit_layers import UnWindowPartitioning
+from keras_cv.layers.maxvit_layers import WindowPartitioning
```

Comment:
Does window partitioning appear in swin transformer? If so, should they place as a separate layers?

Reply:
It does, but afaik, it's a different operation so we couldn't reuse this one potentially.
I'll look into how they differ (if the output differs), since if the output is the same we can use them between them without issues?